kolibri/utils/conf.py

```diff
@@ -54,6 +54,9 @@
     "kolibri_exercise_perseus_plugin"
 ]
 
+#: Everything in this list is added to django.conf.settings.INSTALLED_APPS
+config['INSTALLED_APPS'] = DEFAULT_PLUGINS
```

Comment:
Yes, I think I agree.

You had suggested in previous conversations having a 'blacklist' of disabled plugins, so not just disabling by removing from enabled, but explicitly referencing them as such.

I was willing to make this suggestion here, and defer to long term planning in the conversations that you, @jamalex and @aronasorman are planning to have around settings in general.